src/frameworks/docs/angular.md

```diff
@@ -22,7 +22,7 @@ to Firebase and serve dynamic content to your users.
 <<_includes/_initialize-firebase.md>>
 
 1. Choose your hosting source directory; this could be an existing Angular app.
-1. Choose "Dynamic web hosting with web framework."
+1. Answer yes to "Do you want to use a web framework? (experimental)"
 1. Choose Angular.
```

Comment:
Hmmm... Maybe "If you did not pick a preexisting app in the previous step, choose Angular"? Would want to tag in tech writing on this.